libs/libc/stdio/lib_libvsprintf.c

```diff
@@ -1176,6 +1123,60 @@ static int vsprintf_internal(FAR struct lib_outstream_s *stream,
               break;
 
             case 'p':
+              c = fmt_char(fmt);
+              switch (c)
+                {
+                  case 'V':
+                    {
+                      FAR struct va_format *vaf = (FAR void *)x;
```

Comment:
I see
```
#ifndef CONFIG_LIBC_LONG_LONG
          unsigned long x;
#else
          unsigned long long x;
```
So the question is: Is `unsigned long` is capable to carry full pointer value?

Reply:
Why not? x will be unsigned long long in most arch, it's enough to contain all possible pointer value.